### DISCUSSION TITLE
undiscord is not working when i put index.json

### DISCUSSION THREAD

**Discussion by ghost** [victornpb/undiscord#656] [4. Bugs and Issues]
![Ekran görüntüsü 2024-08-25 191404](https://github.com/user-attachments/assets/42879535-4010-4ef3-91b3-7669fc1b0a70)

**Comment by gaveitatry2** [comment]


**Reply by ghost** [reply]
i mean i already told you i cant delete all of my dms manually it has a lot of messages on different dms and servers and why the developer added that wipe archive thing if its not working it just don't make sense
